keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -210,16 +210,18 @@ metadata:
   finalizers:
   - cosi.io/finalizer [2]
 spec:
-  bucketPrefix: [3]
-  bucketClassName: [4]
-  secretName: [5]
+  requestProtocol: [3]
+  bucketPrefix: [4]
+  bucketClassName: [5]
+  secretName: [6]
 status:
-  bucketContentName: [6]
-  phase: [7]
+  bucketContentName: [7]
+  phase: [8]
   conditions: 
 ```
 1. `labels`: COSI controller adds the label to its managed resources to easy CLI GET ops.  Value is the driver name returned by GetDriverInfo() rpc. Characters that do not adhere to [Kubernetes label conventions](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set) will be converted to ‘-’.
 1. `finalizers`: COSI controller adds the finalizer to defer `Bucket` deletion until backend deletion ops succeed.
+1. `requestProtocol`: specifies the desired data format in which credentials are provided to the app.  One of {“s3”, “gcs”, or “azureBlob”}. 
```

Comment:
What about "swift" and "ceph-rgw"?

Reply:
I think that _azureBlob_ is an accurate name, but in the rpc spec, below, this is referred to as _azureResponse_ rather than _azureBlobResponse_.  FWIW.

Reply:
> What about "swift" and "ceph-rgw"?

Swift will take some more research.  Maybe it can be incorporated during alpha.  We could get some swift guys to take a look, maybe help develop a driver.

> azureBlobResponse

Thanks, will fix.